### PR TITLE
Added link to Balena ADS-B Multi-Service Flight Tracker project

### PR DESCRIPTION
**Pull request by vimondketil**
component: Added link to Balena ADS-B Multi-Service Flight Tracker project. Corrected http to https in URL.

Change-type: patch
Signed-off-by: Ketil Moland Olsen <ketil+balenacommit@ketilmo.com>

### CODE REVIEW

File: pages/learn/more/examples/projects.md [hunk 1]
```diff
@@ -201,6 +201,12 @@ visualization at [123k.de](http://123k.de).
 A dashboard which is visualizing the [Uptime Robot API](https://uptimerobot.com/) on a Raspberry Pi, deployed via balenaCloud.
 This dashboard can be used on TVs or on monitors.
 
+### Balena ADS-B Multi-Service Flight Tracker
+
+[Repository][balena-ads-b] by [ketilmo][ketilmo]
+
+This is project allows you to log humidity and temperature data from a simple sensor and stores that data in a [firebase][firebase] backend.
```
Comment (chrisys) [inline pages/learn/more/examples/projects.md:208]: This description isn't correct!

Comment (ketilmo) [reply pages/learn/more/examples/projects.md:208]: Sorry @chrisys, my mistake. Copy-pasting gone wrong. I will fix this and get back to you.

Comment (vimondketil) [reply pages/learn/more/examples/projects.md]: @chrisys I have corrected the project description now. Feel free to have a look again.

**Review by chrisys — APPROVED** [review]

**Review by vipulgupta2048 — COMMENTED** [review]
@vimondketil Thanks for your PR. Please check the the instructions mentioned in the [Contributing Guide](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) to format your commit in order to merge this PR.

I think the `Change-type: patch` would be appropriate for this change.

**Comment by vimondketil** [comment]
@vipulgupta2048 Thanks for your reply. I have tried to update the pull request per your comments. Please let me know if you need me to change something else.

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
@vimondketil There was a CI/CD check still failing in the change you made, there was an additional footer that was needed to be removed or modified in the first commit you made. It's was a very minor change that was needed to be made. Instead of going through the cycle of changes again, I thought I will probably go in your fork and fix it myself. Here's what I did to help you open your next PR.

- I updated your fork and rebased `patch-1` branch commits above all other commits on master. 
- I then reworded your primary commit to have the right commit template that would pass the checks and also squashed your changes to clean up the PR :100:

Your PR already was approved by @chrisys so balena-CI took care of the merging while I was writing this message. And, that's about it. Again, thanks for contributing - hope the next change gets through faster. We are always working on making the entire process better for everyone involved Cheers! :white_flower: :hatching_chick:

**Comment by vimondketil** [comment]
Thanks, @vipulgupta2048, it's appreciated! And thanks for the detailed explanation. I will try to get it right next time. :)

**Comment by vipulgupta2048** [comment]
Happy to help!

**Pull request merged**